service/models.py

```diff
@@ -72,6 +81,14 @@ class Wishlist(db.Model):
     def __repr__(self):
         return "<Wishlist %r id=[%s]>" % (self.name, self.id)
 
+    @retry(
+        HTTPError,
+        delay=RETRY_DELAY,
+        backoff=RETRY_BACKOFF,
+        tries=RETRY_COUNT,
+        logger=logger,
+    )
+
```

Comment:
i think the convention is not to leave a line between a decorator and the method/function being decorated, please remove the blank lines

Reply:
Nice catch, thanks for the input. I'll modify it.